README.md

```diff
@@ -281,7 +281,7 @@ $ rustup install stable-msvc
 Toolchain names that don't name a channel instead can be used to name
 [custom toolchains].
 
-[MSVC-based toolchain]: https://www.rust-lang.org/downloads.html#win-foot
+[MSVC-based toolchain]: https://www.rust-lang.org/tools/install
```

Comment:
Should we link this to https://forge.rust-lang.org/other-installation-methods.html#stable ?

Reply:
I don't have any suggestions. I just open rust-lang/www.rust-lang.org#726 to request a mean to view Windows installation page without platform auto-detection.

Reply:
If https://github.com/rust-lang/www.rust-lang.org/pull/729 gets merged then https://www.rust-lang.org/tools/install?platform_override=win ought to work.